flight-planner-net/Controllers/AdminController.cs

```diff
@@ -0,0 +1,61 @@
+using FlightPlannerService.Models;
+using FlightPlannerService.Storage;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace FlightPlannerService
+{
+    [Route("admin-api")]
+    [ApiController]
+    [Authorize]
+    public class AdminController : ControllerBase
+    {
+        [Route("flights/{id}")]
+        [HttpGet]
+        public IActionResult GetFlight(int id)
+        {
+            return NotFound();

```

Comment:
?

Reply:
Lūdzu konkrētāk, kas šeit nav pareizi? :)
Mēģināju visādi pamainīt koda daļu šinī funkcijā, bet pēc izmainīšanas bez jebkāda sakara izmet undefined kļūdas par DeleteFlight funkciju un 03. tests neiziet. 
Tā kā laikam atstāšu pagaidām šādi.:)